modules/nw-modifying-operator-install-config.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+
+[id="modifying-nwoperator-config-startup-{context}"]
+= Modifying advanced network configuration parameters
+
+You can modify the advanced network configuration parameters only before you
+install the cluster. Advanced configuration customization lets you integrate
+your cluster into your existing network environment by specifying an MTU or
+VXLAN port, by allowing customization of
+link:https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/[kube-proxy]
+settings, and by specifying a different `mode` for the `openshiftSDNConfig`
+parameter.
+
+[IMPORTANT]
+====
+Modifying the {product-title} manifest files directly is not supported.
+====
+
+.Prerequisites
+
+* Generate the `install-config.yaml` file and complete any modifications to it.
+
+.Procedure
+
+. Optionally, make a backup of your `install-config.yaml` so that you can use it
+to install multiple clusters. The installation program deletes the
+`install-config.yaml` file after generating the manifests.
+. Use the following command to create manifests:
++
+----
+$ openshift-install create manifests --dir=<directory> <1>
+----
+<1> Specify the directory name that contains the `install-config.yaml` file for
+your cluster.
+
+. Create a file named `cluster-network-03-config.yml` in the directory you
+specified:
++
+----
+$ touch <directory>/manifests/cluster-network-03-config.yml <1>
+----
+<1> Specify the directory name that contains the `manifests/` directory for
+your cluster.
++
+After creating the file, a listing of the files in the directory displays the
+following output:
++
+----
+$ ls <directory>/manifests/cluster-network-*
+cluster-network-01-crd.yml
+cluster-network-02-config.yml
+cluster-network-03-config.yml
+----
+
+. Open the `cluster-network-03-config.yml` file in an editor and enter a CR that
+describes the Operator configuration you want:
++
+[source,yaml]
```

Comment:
The way that you've done this confuses me. Why are you showing a sample CR that doesn't match the one that you're including in the assembly and including configuration instructions that don't actually help you configure it?

I think it would be less confusing to drop the correct annotated CR into this module or delete this CR and its annotations and say to see the provided sample CR.

Reply:
Copied to Google Doc to visualize in context.